utility/gmusic_playlists_to_plex.py

```diff
@@ -87,6 +87,17 @@ def compare(ggmusic, pmusic):
     elif title == pmusic.title:
         return [pmusic]
 
+songs = mc.get_all_songs()
```

Comment:
`SONGS = mc.get_all_songs()` and move this to the top under the `mc` assignment.